docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -193,6 +194,14 @@ Request Body:
 
   Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
 
+- `loadbalancer.openstack.org/load-balancer-id`
+
+  This annotation is automatically added to the Service if it's not specified when creating. After the Service is created successfully it shouldn't be changed, otherwise the Service won't behave as expected.  
+
+  If this annotation is specified with a valid cloud load balancer ID when creating Service, the Service is reusing this load balancer rather than creating another one. Again, it shouldn't be changed after the Service is created.
+
+  If this annotation is specified, the other annotations which define the load balancer features will be ignored.
```

Comment:
I think we should error out on such case, not ignore it ..

Reply:
We can't, with this PR, all the services (either existing or new) will have this annotation created automatically, in controller code it's hard to tell if the request is creating new service or updating an existing one.

Reply:
sorry then I am confused.. if we will create this annottation anyway
and you are saying `If this annotation is specified, the other annotations which define the load balancer features will be ignored` , does this mean we will ignore the other annotation in any case? 

anyway, this is not a blocker so we can wait for CI ready then merge and help me in understand the logic, thanks

Reply:
As demonstrated in the doc, if you are creating a new service without this annotation, the workflow is the same as previous, but in the end occm adds this annotation to the service. So next time when you updating this service, occm can directly look for the load balancer by ID rather than searching by name.

However, if you create a service with a valid lb id specified, that means you are just creating listener/pool/members in an existing load balancer, which means you can't config something like fixed/floating IP for the lb, the flavor and az, etc.

Reply:
ok, I understand, maybe my thought is overkill ,so , let it be, thanks for the detailed info~